net/tcp/tcp_sendfile.c

```diff
@@ -148,14 +206,15 @@ static uint16_t ack_eventhandler(FAR struct net_driver_s *dev,
        */
 
       pstate->snd_sent = pstate->snd_acked;
+#ifndef CONFIG_NET_TCP_WRITE_BUFFERS
+      conn->rexmit_seq = pstate->snd_sent + pstate->snd_isn;
+#endif
     }
 
   /* Check for a loss of connection */
 
   else if ((flags & TCP_DISCONN_EVENTS) != 0)
```

Comment:
check the disconnect event at first segment, the conn instance could be NULL:
https://github.com/apache/incubator-nuttx/blob/master/net/netdev/netdev_carrier.c#L97

Reply:
Done. Thank you.